vars/shwrapRc.groovy

```diff
@@ -1,10 +1,13 @@
 def call(cmds) {
-    // If umask is somehow unset, fix it.
+    // default is HOME=/ which normally we don't have access to.
+    // Also if umask is somehow unset, fix it.
+    withEnv(["HOME=${env.WORKSPACE}"]) {
     return sh(returnStatus: true, script: """
```

Comment:
Incorrect indent.

Reply:
Good catch, thanks! Kinda asked for it reviewing the patch with whitespace ignored. :)